api/v1/checks.go

```diff
@@ -527,6 +581,7 @@ func (c NamespaceCheck) GetType() string {
 
 type DNSCheck struct {
 	Description     `yaml:",inline" json:",inline"`
+	ConnectionName  string   `yaml:"connection,omitempty" json:"connection,omitempty"`
```

Comment:
I don't think we need this - DNS servers are not authenticated

Reply:
Ya, that's why I only used `duty.FindConnectionByURL` instead of `duty.HydratedConnectionByURL` because there's no auth. Thought it would still be handy to just get the host from a connection name.


But, i'll remove it.